spec/integration_spec.rb

```diff
@@ -345,12 +347,10 @@ def rollback
 
   context "when an around hook errors early" do
     let(:organizer) {
-      build_organizer(organize: [organizer2, interactor3, organizer4, interactor5]) do
-        around do |interactor|
+      interactors = [organizer2, interactor3, organizer4, interactor5]
+      build_organizer(organize: interactors) do
+        around do |_interactor|
           raise "foo"
-          context.steps << :around_before
-          interactor.call
-          context.steps << :around_after
```

Comment:
Yeah, I see your point. I thought about smth like this:

```ruby
def unexpected_error!
  raise "foo"
end
```

It may be more intention-revealing (the "unexpected" nature), but plain `raise` is far simpler and understable. What do you think?